packages/editor/src/components/parts/program-start/activity/StartPart.test.tsx

```diff
@@ -0,0 +1,69 @@
+import { DeepPartial, render, renderHook, screen } from 'test-utils';
+import { ElementData, InscriptionValidation, ProgramInterfaceStartData } from '@axonivy/inscription-protocol';
+import { PartStateFlag } from '../../../editors';
+import { useProgramInterfaceStartPart } from './StartPart';
+
+const Part = () => {
+  const part = useProgramInterfaceStartPart();
+  return <>{part.content}</>;
+};
+
+describe('StartPart', () => {
+  function renderPart(data?: ProgramInterfaceStartData) {
+    render(<Part />, {
+      wrapperProps: { data: data && { config: data } }
+    });
+  }
+
+  test('empty data', async () => {
+    renderPart();
+    expect(screen.queryByText('Program')).toBeInTheDocument();
+    expect(screen.queryByText('Timeout')).toBeInTheDocument();
+  });
+
+  test('full data', async () => {
+    renderPart({
+      javaClass: 'Test',
+      exceptionHandler: '>> Ignore Exception',
+      timeout: {
+        seconds: '',
+        error: 'ivy:error:program:timeout',
+        action: 'NOTHING',
+        duration: ''
+      }
+    });
+    expect(screen.getByLabelText('Java Class')).toHaveValue('Test');
+  });
+
+  function assertState(expectedState: PartStateFlag, data?: DeepPartial<ProgramInterfaceStartData>, validation?: InscriptionValidation) {
+    const { result } = renderHook(() => useProgramInterfaceStartPart(), {
+      wrapperProps: { data: data && { config: data }, validations: validation && [validation] }
+    });
+    expect(result.current.state.state).toEqual(expectedState);
+  }
+
+  test('configured', async () => {
+    assertState('empty');
+    assertState('configured', {
+      javaClass: 'Bla'
+    });
+
+    assertState('error', undefined, { path: 'javaClass.cause', message: '', severity: 'ERROR' });
+    assertState('warning', undefined, { path: 'exceptionHandler.error', message: '', severity: 'WARNING' });
+  });
+
+  test('reset', () => {
+    let data: DeepPartial<ElementData> = {
+      config: {
+        javaClass: 'Test'
+      }
+    };
+    const view = renderHook(() => useProgramInterfaceStartPart(), {
+      wrapperProps: { data, setData: newData => (data = newData) }
+    });
+    expect(view.result.current.reset.dirty).toEqual(true);
+
+    view.result.current.reset.action();
+    expect(data.config?.javaClass).toEqual('');

```

Comment:
check that the timeout and exceptionHandler configs are also correclty reseted by your code